vector/src/main/res/layout/item_timeline_event_base_state.xml

```diff
@@ -50,7 +56,7 @@
             android:id="@+id/messageVerificationDoneStub"
             style="@style/TimelineContentStubBaseParams"
             android:layout="@layout/item_timeline_event_status_tile_stub"
-            tools:visibility="visible" />
+            tools:visibility="gone" />
```

Comment:
I prefer to keep the visibility for easier debugging/editting.
You could add `tools:layout_marginTop` attribute to ensure the views does not overlap. Also applicable to the other ViewStub.